admin/routesForAdmin3x.json

```
@@ -0,0 +1,10 @@
+{
+    "admin.app.admin-graphiql": {
+        "component": "GraphiQL",
+        "path": "/admin/app/graphiql",
+        "around": ["layout"]
+    },
+    "admin.app.layout": {
```

no need for extending admin.app interface for this one, since it does not have a path
```suggestion
    "layout": {
```

---

it also has to match the value you passed in the "around" array in the other app. So just "layout" fixes it

---

done, thank you @ohmyguigs